libkbfs/keybase_daemon_rpc.go

```diff
@@ -378,6 +378,7 @@ func (k *KeybaseDaemonRPC) OnConnect(ctx context.Context,
 		keybase1.IdentifyUiProtocol(daemonIdentifyUI{k.daemonLog}),
 		keybase1.NotifySessionProtocol(k),
 		keybase1.NotifyUsersProtocol(k),
+		keybase1.FsProtocol(&fsRPC{config: k.config, log: k.log}),
```

Comment:
When you move `fsRPC` into a different package, you'll want to move this protocol registration there too.  You can use `libkbfs.NewSharedKeybaseConnection()` to get a shared connection, I think.